### ISSUE TITLE
Importing keys does not work

### ISSUE THREAD

**Issue by DutchPete** [sparrowwallet/sparrow#944]
I am on macOS 13.3 and have downloaded Sparrow 1.7.4.
When I try to import the keys with `curl https://keybase.io/craigraw/pgp_keys.asc | gpg --import`, I get:

`% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:29 --:--:--     0curl: (6) Could not resolve host: keybase.io
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0`

I have been trying for the past 3 days.
Please advise.

**Comment by craigraw** [comment]
Works fine here - looks like your machine has internet issues if it can't connect to keybase.io. You can get the key using a web browser as well - click on the second (key) field on https://keybase.io/craigraw/.

**Comment by DutchPete** [comment]
Thanks for your quick answer.
Is this the right one?

![image](https://user-images.githubusercontent.com/10794265/232088358-68846083-68a1-4580-8b2e-4569836409b9.png)

If so, I assume the data in the grey block needs to be copy/pasted into a file. What extension should that file have?
Where should it be saved, in the Downloads folder?

I apologize for these basic questions, I am not too familiar with this subject.

**Comment by DutchPete** [comment]
Any chance for a reply?

**Comment by craigraw** [comment]
Yes, this is the correct key. Save as a text file (name is not important). You can the  run the gpg import command shown on the Download page.

**Comment by DutchPete** [comment]
Sorry for my late reaction, I had something urgent I had to attend to.

So, I saved the key as a `.txt` file in Downloads, then ran the import command, but got the same feedback:

`peter@Peters-MacBook-Air Downloads % curl https://keybase.io/craigraw/pgp_keys.asc | gpg --import
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:29 --:--:--     0curl: (6) Could not resolve host: keybase.io
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
peter@Peters-MacBook-Air Downloads %`

What am I doing wrong?
